oaknational/Oak-Web-Application
# Removed wobbly borders on "Child Subjects" section in curriculum visualiser

## Description
Removed wobbly borders on "Child Subjects" section in curriculum visualiser. This was missed in #2908

## Issue(s)

Fixes `CUR-1035`

## How to test

1. Go to https://deploy-preview-2959--oak-web-application.netlify.thenational.academy
2. Check "Child Subjects" doesn't have wobbly borders 
3. Check for regressions of "Child Subjects" buttons
